src/api/src/model.php

```diff
@@ -55,21 +55,21 @@ public static function getValue($pointID, $timestamp){
 	}
 	public static function getValuesByBuildingInRange($buildingID, $start, $end){
 		$db = Database::getInstance();
-		$sth = $db->prepare("SELECT * FROM PointValues 
-			LEFT JOIN Points ON PointValues.PointID=Points.ID
+		$sth = $db->prepare("SELECT PointValues.*, PointTypes.* FROM PointValues 
```

Comment:
I changed this because a lot of the rows are kinda unnecessary for our dashboard and were just clogging up memory, but if we find that we need them then can change again. If not, then can change the getValuesByBuildingInRangeByType select as well